encryption/saber/saberKEM-sha3/param/LightSABER.yaml

```diff
@@ -0,0 +1,14 @@
+name: LightSABER
+security level:
+  classical: 125
+  quantum: 114
+  nist category: 1
+  comment: 'See https://www.esat.kuleuven.be/cosic/pqcrypto/saber/index.html'
```

Comment:
I don't understand why you added this extra note?

Reply:
So, I should've added more information, since this is kind of confusing and I don't know if we should add additional third party information.

While adding SABER I compared the information from the homepage with the submission paper. And the security numbers are off. 
So [here](https://estimate-all-the-lwe-ntru-schemes.github.io/docs/) are security estimations about LWE/NTRUE schemes. And these numbers are shown on the SABER homepage (but not all correctly). These are other assumption numbers than in the submission paper. This is also true for other schemes that we already included, but the numbers still don't match all the time (even the claimed security).

Long story short I think I should add the security bits from the original submission paper and delete the other values. This is also consistent with the other schemes.

Reply:
About those comic sans ms ;) values: agreed that we should not add those if they differ.

About the paper I linked: I'd say it's not really external (two common authors) so I'd say we can and should include it.

Reply:
I agree that we should include the paper that you linked. However, I'm not sure how useful the data is for comparison. The paper includes only timings for the decapsulation function for one parameter set. So we are missing key generation and encapsulation, and two parameter sets.
They're writing that the benchmarks are included in pqm4, but I couldn't find them.

For now, I would include SABER without additional data. We can have a separate look, or the data still shows up in pqm4.

Reply:
> The paper includes only timings for the decapsulation function for one parameter set

ah, ok, I didn't look into it that deep. Agreed, then.

PQM4 is a separate issue anyways. I'll re-run the imports next week or so when we merged everything.